ui/app/styles/components/linkable-item.scss

```diff
@@ -10,12 +10,3 @@
   grid-row: 1;
   align-self: center;
 }
-
-.overflow-ellipsis {
```

Comment:
Moved to `read-only.scss` once we refactored `LinkableItem` into separate components and created  a `ReadOnly` component - forgot the remove styling from this `.scss` file